webapp/channels/src/components/sidebar/sidebar_category/sidebar_category_menu/sidebar_category_menu.tsx

```diff
@@ -264,6 +285,8 @@ const SidebarCategoryMenu = (props: Props) => {
                     onToggle: handleMenuToggle,
                 }}
             >
+                {markAsReadMenuItem}
```

Comment:
should this menu be always present even when category has no unread messages?

Reply:
But i believe it would be extra cost to loop over the categories channels and get the unreads counter. right ?

Reply:
Up to @abhijit-singh . If we make it completely dynamic, my fear is the menu appearing suddenly when you are about to click something else because some channel got unread. So if we were to do something like this, I would keep it "static". Once you open the menu, you won't show any new items. The problem with that is that users may get confused on why the button sometimes appear and sometimes does not.

Reply:
yup its fine to leave as is